widgets/common/src/RemoveDeploymentModal.jsx

```diff
@@ -0,0 +1,61 @@
+function RemoveDeploymentModal({ open, deploymentId, force, onHide, toolbox }) {
+    const {
+        Basic: { Confirm, ErrorMessage },
+        Common: { DeploymentActions },
+        Hooks: { useErrors, useOpenProp },
+        i18n
+    } = Stage;
+
+    const { errors, setErrors, clearErrors } = useErrors(null);
+
+    useOpenProp(open, clearErrors);
+
+    const content = i18n
+        .t(`widgets.common.deployments.removeModal.${force ? 'forceDelete' : 'delete'}Description`, { deploymentId })
+        .split('\n')
+        // eslint-disable-next-line react/no-array-index-key
+        .map((line, index) => <p key={index}>{line}</p>);
+
+    function deleteDeployment() {
+        const actions = new DeploymentActions(toolbox);
+        const deleteAction = force ? 'doForceDelete' : 'doDelete';
+
+        clearErrors();
+        actions[deleteAction]({ id: deploymentId })
```

Comment:
Few characters more, but it will be easier to trace function calls:
```suggestion
        const deleteAction = force ? actions.doForceDelete : actions.doDelete;

        clearErrors();
        deleteAction({ id: deploymentId })
```

Reply:
Are you sure it will work? 

If I remember correctly I tried the same solution and it was not possible to use it this way.

Reply:
> Are you sure it will work?

If you are asking then I'm almost sure it won't 🙂.

OK, lets leave it.